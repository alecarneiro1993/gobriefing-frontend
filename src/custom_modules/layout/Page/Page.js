import React from "react";
import { Container } from "@material-ui/core";
import { StylesProvider } from "@material-ui/styles";
import styled from "styled-components";

const PageWrapper = styled(Container)`
  display: flex;
  align-items: center;
  padding: 0;
`;

export default function Page(props) {
  return (
    <StylesProvider injectFirst>
      <PageWrapper {...props} component="main" />
    </StylesProvider>
  );
}
