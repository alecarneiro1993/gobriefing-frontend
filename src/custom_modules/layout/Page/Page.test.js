import React from "react";
import { shallow } from "enzyme";

import Page from "./Page";

describe("PageContainer - the main container for pages", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<Page />);
    expect(wrapper).toMatchSnapshot();
  });
});
