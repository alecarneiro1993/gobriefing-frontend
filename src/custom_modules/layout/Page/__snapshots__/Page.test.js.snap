// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContainer - the main container for pages renders without crashing 1`] = `
<StylesProvider
  injectFirst={true}
>
  <Styled(WithStyles(ForwardRef(Container)))
    component="main"
  />
</StylesProvider>
`;
